io_mesh_w3d/w3d/export_w3d.py

```diff
@@ -26,21 +26,26 @@ def save(self, export_settings):
     hlod = create_hlod(hierarchy, containerName)
 
     if export_mode in ('M', 'HAM'):
+        meshes = retrieve_meshes(self, hierarchy, rig, containerName)
+        if not meshes:
```

Comment:
It would probably make sense to not propose these export modes when there is no mesh inside the scene

Reply:
i think completely removing the specific export modes in those cases would confuse the user. Dont know if there is a way to disable it and Show a Tooltip with the Information why it is disabled. But this would also require to retrieve all relevant data before the export view is displayed. So if a user just wants to export mesh or hierarchy it would take a lot of time to show the Export view if he has a complex animation in the scene.

Reply:
I don't think it would be very difficult/expensive to check if there's any meshes/rigs inside the scene with the blender api, which would already make it a lot easier for users to export the correct data.

Reply:
Yes its definetly possible, but would require to Change the Export modes on the fly, which I need to further investigate if it is possible and how. Should we create an issue for that or just take care of it in a feature PR?

Reply:
I'd suggest a seperate issue for that, which can be solved at a later state